README.md

```diff
@@ -1,107 +1,59 @@
-
-    Light_source: ["UV", "Solar", "UV-Vis", "Monochromatic", "Solar Simulator"]
-
-    Lamp: ["Fluorescent", "Mercury" ,"Halogen" ,"Mercury-Xenon", "LED", "Tungsten", "Xenon","Tungsten-Halide", "Solar Simulator"]
-
-    Please generating restrictively follow the format, and must start the generation as the format. Do not generate anything else.
-
-    Light_source: XXX
-
-    Lamp: XXX
-
-### Run Generation
-
-`Groq.ipynb` contains all the libraries needed as well as the code to run the experiment. The needed data files for run the model and evaluate the model contains in the folder `Solar/ground_truth_data/`.
-
-## Evaluation
-
-### Basic Evaluation Process
-- We adopt the model from [Massive Text Embedding Benchmark](https://huggingface.co/blog/mteb) based on the STS Task to calculate the similarity between each generate term and corresponding term in the ground truth. [Model ID](https://huggingface.co/Salesforce/SFR-Embedding-Mistral)
-- We apply the similarity to each term pair (ground_truth_term, generation_term)
-- In case of the number of generated terms and ground truth mismatched, we take the minimal number of generation and ground truth as the number of term we evaluate.
-- We set the threshold as 0.85 for correct generation, which 1 indicate correct generation and 0 indicate miss generation
-- Then calculate the overall accuracy for each item.
-
-### Rule-based Evaluation Process
-
-- If the ground_truth is in the generation, we consider it is accurate
-- If generated reactor type is `Optical Fiber` or `Packed-bed` or `Membrane` and the ground truth is `Fixed-bed`, we consider it is accurate
-- Then apply the same evaluation process as `Basic Evaluation Process`
-
-
-
-### Evaluation Leadboard
-
-| Rank |   Model  | Catalyst | Co-Catalyst | Light Source | Lamp   | Reactor Type | Reaction Medium | Operation Mode |  Overall Average |
-|------|------------|----------|-------------|--------------|--------|--------------|-----------------|----------------|----------------|
-|1|LLama_3.1_70B|1.000|0.9655|0.8966|0.8966|0.8621|0.0345|0.9655|0.8030|
-|2|Claude_3_opus|0.9655|0.6552|0.7241|0.7586|0.5517|0.6897|0.8966|0.7488|
-|3|LLama_3_70B|0.8276|0.6207|0.8276|0.6207|0.3103|0.6207|0.8276|0.665|
-|4|GLM4|0.931|0.4828|0.5517|0.4828|0.5517|0.6897|0.8966|0.6552|
-|5|GPT4|0.8621|0.4828|0.6552|0.6552|0.4483|0.6207|0.8621|0.6552|
-|6|LLama_3.1_8B|0.8276|0.5862|0.5172|0.4138|0.4828|0.6897|0.8276|0.6207|
-|7|LLama_3_8B|0.7931|0.4828|0.5172|0.5172|0.3103|0.6552|0.7241|0.5714|
-|8|GPT3.5|0.6552|0.4138|0.7241|0.3448|0.3448|0.5172|0.8966|0.5567|
-
-
-
-
-
+# Solar-QA-CLI
+This repository contains the command-line tool for [solar-qa pipepine](https://github.com/oeg-upm/solar-qa-eval)
+
+## Requirement
+
+### Requirement for Paper Extraction
+
+- [Grobid](https://grobid.readthedocs.io/en/latest/)
+
+### Requirement for Generation
+
+All required libraries are detailed in requirement.txt. 
+
+```console
+pip install -r requirement.txt
+```

```

Comment:
I think the requirements file is not valid. It's usually something like `bs4==0.0.1` with one line per lib. 
Have you tested this in a clean environment?